src/implementation/Client/DaprClient.ts

```diff
@@ -96,10 +94,6 @@ export default class DaprClient {
       throw new Error('DAPR_INCORRECT_SIDECAR_PORT');
     }
 
-    if (String(SDK_PACKAGE_NAME) === "dapr-client") {
```

Comment:
Was it intentionally removed?

Reply:
No, must be a merge, I will re-add it :)